runit-elixir/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM socrata/runit
+
+# Set up environment
+ENV LANG C.UTF-8
+
+# Add erlang solutions repo
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update &&\
+    DEBIAN_FRONTEND=noninteractive apt-get -y install wget &&\
+    wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb &&\
+    DEBIAN_FRONTEND=noninteractive dpkg -i erlang-solutions_1.0_all.deb
```

Comment:
This might be formatting we use elsewhere, but would you be okay putting the `&&`s at the beginning of the line and/or adding a space in front of the `\`.